webapp/src/routes/app/+page.svelte

```diff
@@ -1,53 +1,221 @@
+            <ul class="dropdown">
+                {#each results as result}
+                    <!-- svelte-ignore a11y-click-events-have-key-events a11y-no-noninteractive-element-interactions -->
+                    <li on:click={() => handleSelect(result)}>
+                        {result.properties.formatted}
+                    </li>
+                {/each}
+            </ul>
+        {/if}
+
+        {#each spots_data as listing}
+            <div
+                class="booking-info-container {listing.id === selectedListingId ? 'highlight' : ''}"
+                id={`listing-${listing.id}`}
+            >
+                <SpotsListComponent {listing} />
+            </div>
+        {/each}
+
+        {#if showBackToTop}
+            <button class="back-to-top" on:click={scrollToTop}> ↑ Back to Top </button>
+        {/if}
+    </div>
+
+    <div class="map-view">
+        <MapLibre
+            center={mapCenter}
+            {zoom}
+            style="https://basemaps.cartocdn.com/gl/positron-gl-style/style.json"
+        >
+            <GeolocateControl position="bottom-left" fitBoundsOptions={{ maxZoom: 12 }} />
+            {#each spots_data as { id, location }}
+                <DefaultMarker
+                    lngLat={[location.longitude, location.latitude]}
+                    on:click={() => handleMarkerClick(id)}
+                >
+                    <Popup offset={[0, -10]}>
+                        <div class="text-lg font-bold">
+                            {location.street_address}, {location.city}
+                        </div>
+                        <button on:click={() => handleMarkerClick(id)}>Go to listing</button>
+                    </Popup>
+                </DefaultMarker>
+            {/each}
+        </MapLibre>
     </div>
-</body>
+</div>
 
 <style>
     .container {
         display: flex;
         flex-direction: row;
-        margin: 8.5rem 3rem;
         max-height: fit-content;
     }
 
     .listings {
         display: flex;
-        width: 30%;
-        min-height: 100%;
+        width: 25%;
+        max-height: 82vh;
+        overflow-y: auto;
         flex-direction: column;
-        font-size: 1.2rem;
-        font-weight: bold;
-        align-items: center;
-        justify-content: center;
+        padding: 0.5rem;
         background-color: rgb(186, 214, 183);
+        position: fixed;
+        z-index: 999;
+        border: 1px solid rgb(104, 104, 104);
+        border-radius: 5px;
     }
 
-    .map-view {
-        width: 75%;
+    .dropdown {
+        position: absolute;
         background-color: white;
+        border: 1px solid #868686;
+        width: 100%;
+        max-height: 10rem;
+        overflow-y: auto;
+        z-index: 1000;
+        margin-top: 3rem;
+    }
+    .dropdown li {
+        padding: 0.5rem;
+        cursor: pointer;
+    }
+    .dropdown li:hover {
+        background-color: #f0f0f0;
+    }
+
+    @keyframes smoothBlink {
```

Comment:
Can you make the blinking more visible. maybe use some color other than green and set opacity to 0.7 instead of 0.5. I can not really see it on the page